Documentation/Installation/Index.rst

```diff
@@ -0,0 +1,52 @@
+..  include:: /Includes.rst.txt
+
+..  _installation:
+
+=============
+Installation
+=============
+
+This project is not a TYPO3 extension, but a standalone application
+used to render documentation. If you want to learn more about how to write
+documentation, please check the TYPO3 documentation website.
```

Comment:
```suggestion
documentation, please check the :ref:`Contribtuib Guide - How to Document <2document:contribute>`
```

You also need to add "h2document     = https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/"  in the guides.xml to the interlink mapping

Reply:
(Typo in "Contribtuib" -> "Contributing")